contrib/nitiwiki/src/wiki_base.nit

```diff
@@ -194,9 +194,6 @@ class Nitiwiki
 		end
 		var file = expand_path(config.root_dir, config.templates_dir, name)
 		var tpl = new TemplateString.from_file(file)
-		if tpl.has_macro("ROOT_URL") then
-			tpl.replace("ROOT_URL", config.root_url)
```

Comment:
I didn't think about this one but in fact, it can really be usefull in the template when you want to add some js or piwik script.